### DISCUSSION TITLE
Contribute to rust port

### DISCUSSION THREAD

**Discussion by nldvos** [rizsotto/Bear#585] [General]
I use bear regularly in my current work project. This is an interesting project. Browsing the repository, I saw that a rust port is in the making. This interests me and, if possible, I would like to contribute to this remake. Can I get directions to know where help is needed?

**Comment by rizsotto** [comment]
Hi @nldvos , thanks for your message. I am happy to have contributions to the rust port. At the moment I am figuring out the project structure and the content. (Mostly reading old and new problem reports to find out what the weak points of the current design.) Will come up with the user documentation first and then we can jump some implementation. The current source files are mostly to discover what is possible to achieve with this new language and the available libraries.

**Comment by rizsotto** [comment]
Since there are interest in the rewrite I am trying to set up a contribution plan.

- will create tickets and [project board](https://github.com/users/rizsotto/projects/4/views/2)
- will allocate some time on the weekends (regularly) to review PRs and update tickets
